packages/sdk-snippets/src/targets/dotnet/net6/webhooks/client.ts

```diff
@@ -60,13 +60,16 @@ export const net6: Client = {
     if (server.length) {
       push('// OAS Server variables', 2);
       server.forEach(data => {
-        push(
+        pushVariable(
           `${escapeForObjectKey(data.name, true)} = "${escapeForDoubleQuotes(
```

Comment:
I looked into trying to refactor this escaping logic into the `pushVariable` method but couldn't come up with a clean way to declare that an object key in .NET needs to be wrapped in quotes, where as one in Node might not need to be. Would be nice to refactor this work at some point down into that codebuilder, so that this work is handled automatically everywhere, but I'm leaving this as-is until I can think of a clean way to do it.